.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
```

Comment:
- [ ]  Please commits should be in the present tense like add linter, add h1 tag, or something like this
![image](https://user-images.githubusercontent.com/74744787/184949180-90e85bdc-0247-44e6-8dc4-62f700893bf1.png)

Reply:
Alright, do you want me to add new commits or what to do to resolve the issue ?